contracts/CarbonVoteX.sol

```diff
@@ -0,0 +1,178 @@
+pragma solidity ^0.4.23;
```

Comment:
use Solium  in https://github.com/ventureum/Documents/wiki/Style-Guide to reformat your codes